R/convert.R

```diff
@@ -13,37 +13,34 @@ rtf_to_html <- function(file) {
     stop("Input RTF file does not exist: ", file)
   }
 
-  if (Sys.which("pandoc") == "") {
-    stop(
-      "Pandoc command not found. ",
-      "Please ensure Pandoc is installed and in your system's PATH."
-    )
-  }
+  # Check if pandoc is available - if not, let pandoc package handle it
+  # The pandoc package will automatically handle installation when needed
```

Comment:
Where did you see that the pandoc package will automatically handle installation? It looks like `pandoc_convert()` calls `pandoc_run()` which will throw an error if pandoc can't be found: https://github.com/cderv/pandoc/blob/45960a34000526a043178cea0482ea6689fa7b5c/R/pandoc-run.R#L19